crates/red_knot_project/src/metadata.rs

```diff
@@ -551,6 +589,176 @@ expected `.`, `]`
 
         Ok(())
     }
+    #[test]
+    fn requires_python_no_python_version() -> anyhow::Result<()> {
```

Comment:
Why does this have a `_no_python_version` suffix, while the other tests below do not? Remove the suffix? Maybe call this
```suggestion
    fn requires_python_major_minor() -> anyhow::Result<()> {
```